egs/librispeech/asr1/run.sh

```diff
@@ -268,7 +268,11 @@ if [ ${stage} -le 5 ]; then
         done
 
         #### use CPU for decoding
-        ngpu=0
```

Comment:
Similar comment to the above.
I think it's better to use cpu for the original setup.
How about making a variable `rec_ngpu` in the beginning of the script, and we can change it in run.sh e.g., `./run.sh --stage 5 --rec_ngpu 1`